client/src/App.tsx

```diff
@@ -2,8 +2,8 @@ import { createBrowserRouter, RouterProvider } from "react-router-dom";
 import Home from "./pages/Home.tsx";
 import AppShell from "./components/AppShell.tsx";
 import ErrorElement from "./pages/ErrorElement.tsx";
-import Challenges from "./pages/Challenges.tsx";
-import Challenge from "./pages/Challenge.tsx";
+import Challenge from "./pages/Challenge.tsx"; //TODO: REMOVE THIS LINE
+import Profile from "./pages/Profile.tsx";
```

Comment:
Might cause a merge conflict. Maybe try to `rebase main` in your branch before merging, so that u can resolve conflict safely. Let me know if need help rebasing.